e2e_tests/test_browse.py

```diff
@@ -55,14 +65,18 @@ def test_browse_filter_functionality_with_query_string_parameters(
         header_rows = aau_user_page.locator(
             "#tbl_result tr:visible"
         ).evaluate_all(
-            """els => els.slice(0).map(el =>
-              [...el.querySelectorAll('th')].map(e => e.textContent.trim())
-            )"""
+            """els => els.map(el => {
+                var headerCells = [...el.querySelectorAll('th')];
+                headerCells.shift();
+                return headerCells.map(e => e.textContent.trim());
```

Comment:
why headerCelss.shift() has been used, can you not use the class to identify a row similar to the implementation in e2e_tests/test_search.py

rows = aau_user_page.locator("#tbl_result tr.top-row").evaluate_all(
            """els => els.slice(0).map(el =>
              [...el.querySelectorAll('td.search__mobile-table__top-row')].map(e => e.textContent.trim())
            )"""
        )

Reply:
.shift has been removed.

Reply:
resolved on call